src/validation/user.validation.js

```diff
@@ -76,5 +82,11 @@ const validateResult = (req, res, next) => {
 };
 
 export default {
-  checkFirstName, checkLastName, checkValidEmail, checkExistingEmail, checkPassword, validateResult
+  checkFirstName,
```

Comment:
```suggestion
  checkFirstName,
```
Would you please remove below empty space?